### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by jwodder**
Part of dandi/dandi-cli#170

### CODE REVIEW

File: serve.py [hunk 1]
```diff
@@ -6,6 +6,10 @@
 from sanic_cors import CORS
 import requests
 
+GIRDER_URL = os.environ.get("GIRDER_URL", "https://girder.dandiarchive.org").rstrip('/')
+GUI_URL = os.environ.get("GUI_URL", "https://gui.dandiarchive.org").rstrip('/')
+REDIRECTOR_URL = os.environ.get("REDIRECTOR_URL", "https://dandiarchive.org").rstrip('/')
```
Comment (satra) [inline serve.py]:
```suggestion
ABOUT_URL = os.environ.get("ABOUT_URL", "https://www.dandiarchive.org").rstrip('/')
```

File: serve.py [hunk 2]
```diff
@@ -86,32 +90,32 @@ async def init(app, loop):
...
 @app.route("/", methods=['GET'])
 async def main(request):
-    return response.redirect("https://gui.dandiarchive.org/")
+    return response.redirect(GUI_URL + '/')
 
 
 @app.route("/about", methods=['GET'])
 async def about(request):
-    return response.redirect("https://www.dandiarchive.org")
+    return response.redirect(REDIRECTOR_URL)
```
Comment (satra) [inline serve.py]:
```suggestion
    return response.redirect(ABOUT_URL)
```

**Review by satra — CHANGES_REQUESTED** [review]
the www.dandiarchive.org is served from github pages. so it's not the redirectory url which is dandiarchive.org

File: serve.py (hunk 1 above)
Comment (jwodder) [reply serve.py]: Variable name changed.

Comment (satra) [reply serve.py]: the `www` is important as well.

Comment (jwodder) [reply serve.py]: Added.

**Comment by satra** [comment]
thanks @jwodder - while we are at this, it could be useful to have a github action test here that we can run this script and check for the responses with and without these variables? since the redirector no longer actually queries the service, this should be much more straightforward now. 

we could also check the docker build as well. i leave it to you and @yarikoptic whether its done now or in a future PR. otherwise this LGTM.

**Comment by yarikoptic** [comment]
> we could also check the docker build as well. i leave it to you and @yarikoptic whether its done now or in a future PR. otherwise this LGTM.

I like tests! We should hangout!  (ref: [Idiocracy](https://www.imdb.com/title/tt0387808/) although not about tests specifically)

@satra -- are these changes go "hot" automagically right upon merge or it requires manual "update" on your end ATM?  If go "hot" -- we better add tests right away.  Otherwise - IMHO ok to have a separate issue PR.  But I will leave it to @jwodder to decide overall.

**Comment by jwodder** [comment]
Tests do seem like a good idea, but out of scope for this PR.

**Comment by yarikoptic** [comment]
Filed separate #17 so we do not forget

**Comment by satra** [comment]
> are these changes go "hot" automagically

@yarikoptic - nope. it's a manual restart at the moment. but i'm hoping this will get added to the terraform setup.

**Pull request merged**